ga4gh/datamodel/variants.py

```diff
@@ -539,15 +585,15 @@ def getVariants(self, referenceName, startPosition, endPosition,
         if variantName is not None:
             raise exceptions.NotImplementedException(
                 "Searching by variantName is not supported")
-        if callSetIds is not None:
-            raise exceptions.NotImplementedException(
-                "Specifying call set ids is not supported")
+        if callSetIds is None:  # This will get all Variants
```

Comment:
@shajoezhu just so you know, an email gets sent out to everyone that is subscribed to a thread (like this PR) every time anyone makes a comment on it.  I would advise leaving out these "check" comments.  It doesn't really add anything to the discussion for anyone else, and we presume you see the comments already!